splink/linker.py

```diff
@@ -3446,6 +3446,7 @@ def cluster_studio_dashboard(
         cluster_names: list = None,
         overwrite: bool = False,
         return_html_as_string=False,
+        _df_cluster_metrics: SplinkDataFrame = None,
```

Comment:
The docstring below for `sampling_method (str, optional)` should add the new option.

I think we might also want to name this a bit differently.  If I understand the code, it's not sampling by density, it's finding the lowest density clusters.

To further explain the distinction, sampling  `by_cluster_size` performs a stratified sample that picks `n` clusters of each size.  (i.e. it doesn't just pick the largest clusters)

So (if I'm right that this PR takes the largest clusters, rather than performs a stratified sample by cluster), this argument should be something like `lowest_density_clusters`.

Alternatively we may wish to change the sampling logic so that it does sample clusters of different sizes

Reply:
Thanks for your comments Robin! I agree with your name change suggestion and have updated to `lowest_density_clusters` in the docstring and `cluster_studio.py`

I’ve been giving some thought as to what would be most useful from a user’s perspective and I think there could be more benefit to performing stratified sampling, either by cluster size or cluster density (although for the latter some thought would be needed on how to partition density i.e. what size buckets to use)

I’ll post on the Slack channel to get input on the different options I’ve thought about so far with the view to implement suggestions as part of a separate PR building on from this one

Reply:
Yep, agreed.  Happy with that approach